### DISCUSSION TITLE
Installing version 2.0

### DISCUSSION THREAD

**Discussion by RBstudentPhD** [neperfepx/FEPX#113] [Q&A]
Hi,

I was using FEPX on a calculation server (Compute Canada).
In October I was able to run the software without issue.
Now I'm trying to run the version 2.0 by reproducing the same methodology, but I'm having trouble to do so.
In the zip file you can find the commands I used to compile FEPX as well as my simulations files.
Everything seems ok when compiling, but all the tests failed as well as my simulation (they were also failing in October but my simulations were still running).
[Files2.zip](https://github.com/neperfepx/FEPX/files/14255664/Files2.zip)

I also tryed to install it on my personal linux desktop, but I'm having trouble with installing openmpi :
```
  CMake Error in /home/XXXX/Documents/FEPX/src/build/CMakeFiles/CMakeTmp/CMakeLists.txt:
    Imported target "MPI::MPI_Fortran" includes non-existent path
      "/usr/lib/x86_64-linux-gnu/fortran/gfortran-mod-15/openmpi"
    in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
    * The path was deleted, renamed, or moved to another location.
    * An install or uninstall procedure did not complete successfully.
    * The installation package was faulty and references files it does not
    provide.
```

I did install openmpi using brew :
 ```
 $ brew install openmpi
    Running `brew update --auto-update`...
    ==> Auto-updated Homebrew!
    Updated 2 taps (homebrew/core and homebrew/cask).
    ==> New Formulae
    ktfmt                                            moon
    ==> New Casks
    ollamac
    You have 1 outdated formula installed.
    Warning: open-mpi 5.0.1 is already installed and up-to-date.
    To reinstall 5.0.1, run:
      brew reinstall open-mpi
```

I'm not an expert with installing libraries on Linux unfortunately :/
Any help from the community would be greatly appreciated :)
Thank you in advance,
Regards,
RB

**Comment by mkasemer** [comment]
With regard to dependencies like compilers and MPI, I would suggest talking to your system administrators to assist you with installation and configuration. This is highly system dependent, and as such is difficult to debug and arrive at a clean response.

I would also suggest waiting for the next release, within a week or two, as we have found some bugs that need to be addressed in the new version.
